lib/matplotlib/axes/_base.py

```diff
@@ -1261,43 +1261,51 @@ def get_aspect(self):
-           num       a circle will be stretched such that the height
-                     is num times the width. aspect=1 is the same as
-                     aspect='equal'.
-          ========   ================================================
-
-        *adjustable*
-
-          ============   =====================================
-          value          description
-          ============   =====================================
-          'box'          change physical size of axes
-          'datalim'      change xlim or ylim
-          'box-forced'   same as 'box', but axes can be shared
-          ============   =====================================
+        Set the aspect of the axis scaling, i.e. the ratio of y-unit to x-unit.
 
-        'box' does not allow axes sharing, as this can cause
-        unintended side effect. For cases when sharing axes is
-        fine, use 'box-forced'.
-
-        *anchor*
+        Parameters
+        ----------
+        aspect : ['auto' | 'equal'] or num
+            Possible values:
+
+            ========   ================================================
+            value      description
+            ========   ================================================
+            'auto'     automatic; fill position rectangle with data
+            'equal'    same scaling from data to plot units for x and y
+             num       a circle will be stretched such that the height
+                       is num times the width. aspect=1 is the same as
+                       aspect='equal'.
+            ========   ================================================
+
+        adjustable : None or ['box' | 'datalim' | 'box-forced'], optional
+            If not ``None``, this defines which parameter will be adjusted to
+            meet the required aspect. See `.set_adjustable` for further details.
+
+        anchor : None or str or 2-tuple of float, optional
+            If not ``None``, this defines where the axes will be drawn if there
+            is extra space due to aspect constraints. The most common way to
+            to specify the anchor are abbreviations of carindal directions:
+
+            =====   =====================
+            value   description
+            =====   =====================
+            'C'     centered
+            'SW'    lower left corner
+            'S'     middle of bottom edge
+            'SE'    lower right corner
+            etc.
+            =====   =====================
+
+            See `.set_anchor` for further details.
 
-          =====   =====================
-          value   description
-          =====   =====================
-          'C'     centered
-          'SW'    lower left corner
-          'S'     middle of bottom edge
-          'SE'    lower right corner
-          etc.
-          =====   =====================
+        See Also
+        --------
+        matplotlib.axes.Axes.set_adjustable
```

Comment:
Do these need to be made as links?

Reply:
Not needed https://4409-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/api/_as_gen/matplotlib.axes.Axes.set_aspect.html#matplotlib.axes.Axes.set_aspect

Reply:
Thanks! Deleting comments below just to keep things clean...